docs/release/new-release-series.md

```diff
@@ -31,6 +31,20 @@ Prepare Koji and OSG-Build
 
     -   Apply the Koji Ansible config on the Koji Hub host.
 
+            :::console
```

Comment:
`--check` is 'dry-run mode' meaning it lets you preview changes without making them; you should add something about running that command again without `--check`.

Reply:
ok how about now?

Reply:
Yep, looks good.